extensions/wikia/ArticleVideo/templates/ArticleVideo_featured.php

```diff
@@ -15,6 +15,7 @@
 			<div class="video-label"><?= wfMessage('articlevideo-watch')->escaped() ?><span class="video-time"></span></div>
 			<div class="video-title"></div>
 		</div>
+		<?= $app->renderView( 'ArticleVideo', 'feedback' ) ?>
```

Comment:
I tried to use it but I got an exception. It looks for `ArticleVideoController_feedback.php` file instead of `ArticleVideo_feedback.php`. I wouldn't like to rename that file.